src/dragonball/src/vcpu/vcpu_impl.rs

```diff
@@ -273,7 +273,7 @@ enum VcpuEmulation {
 /// A wrapper around creating and using a kvm-based VCPU.
 pub struct Vcpu {
     // vCPU fd used by the vCPU
-    fd: Arc<VcpuFd>,
+    pub(crate) fd: Arc<VcpuFd>,
```

Comment:
maybe we can add a function `get_vcpu_fd()`?

Reply:
Thanks, @YushuoEdge! I will fix this in next push.